lib/LaTeXML/Engine/Base_ParameterTypes.pool.ltxml

```diff
@@ -360,19 +360,23 @@ sub readDigested {
   no warnings 'recursion';
   my ($gullet) = @_;
   $gullet->skipSpaces;
-  my $ismath = $STATE->lookupValue('IN_MATH');
-  my @list   = ();
+  my $ismath  = $STATE->lookupValue('IN_MATH');
+  my @list    = ();
+  my $grouped = 0;
   my $token;
   do { $token = $gullet->readXToken(0);
   } while (defined $token && (($token->getCatcode == CC_SPACE) || $token->equals(T_CS('\relax'))));
   if    (!defined $token) { }
   elsif ($token->getCatcode == CC_BEGIN) {
+    $grouped = 1;
     Digest($token);
     push(@list, $STATE->getStomach->digestNextBody()); pop(@list); }    # content w/o the braces
   else {
     push(@list, $STATE->getStomach->invokeToken($token)); }
   @list = grep { ref $_ ne 'LaTeXML::Core::Comment' } @list;
-  return List(@list, mode => ($ismath ? 'math' : 'text')); }
+  my $list = List(@list, mode => ($ismath ? 'math' : 'text'));
+  $list->setProperty('grouped', $grouped) if $grouped;
+  return $list; }
```

Comment:
Ugh! Adding a `grouped` property seems really klunky, but hard to see the proper alternative...

Reply:
Thinking of code precedent... The Whatsit `revert` method already uses certain properties to make decisions, so I could make this technique "more official" by enhancing List's `revert` method?

That would avoid the appearance of a special case, and any List with that property set would then revert braced.

Reply:
It's still a special case, just more generic :> But maybe `braced` is better, w/o the TeX-group implication? ORR since the braces is really an illusion anyway, maybe the 2nd "(" should just fall outside of the `\left` anyway, so then there's no braces in the reversion in any case? (which would amount to `TeXDelimiter` needing it's own version of `readDigested`?)